package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "types": "index.d.ts",
   "bin": {
-    "@fastify/secure-session": "genkey.js"
+    "secure-session-gen-key": "./bin/generate-key.js"
```

Comment:
It is not resolving the issue. The ultimate goal is using `npx @fastify/secure-session`

Reply:
tbh I dont want to handle the issue in the npm/yarn codebase.